packages/angular/src/builders/module-federation-dev-server/schema.json

```
@@ -118,6 +118,10 @@
         "type": "string"
       },
       "description": "List of remote applications to not automatically serve, either statically or in development mode. This can be useful for multi-repository module federation setups where the host application uses a remote application from an external repository."
+    },
+    "pathToManifestFile": {
+      "type": "string",
+      "description": "Path to a module-federation.manifest.json file containing the dynamic remote applications relative to the workspace root."
```

I think we need to provide more info about the file structure, either inline or pointing to another link if we have it, but we can leave that for a follow-up task. It's not an issue of this option only, some of our schema options need a good review and rewording.

I do think we shouldn't explicitly say `module-federation.manifest.json` because folks might interpret that it must have that name and the option is just about the location. Maybe something like:
```suggestion
      "description": "Path to a Module Federation manifest file (e.g. `my/path/to/module-federation.manifest.json`) containing the dynamic remote applications relative to the workspace root."
```